src/Microsoft.DotNet.Interactive.Tests/CompositeKernelTests.cs

```diff
@@ -525,6 +526,28 @@ public void When_child_kernel_is_added_then_its_frontend_environment_is_obtained
                 .BeSameAs(compositeKernel.FrontendEnvironment);
         }
 
+        [Fact]
+        public async Task When_kernel_implements_IKernelCommandHandler_for_a_custom_command_it_is_correctly_selected_has_handler()
```

Comment:
An interesting test case that I'm not sure if we have: What if there's an implemented `IKernelCommandHandler<Blah>` and then someone registers a custom handler for `Blah`? I think this should override the declared one. Do you know if this works?